docs/getting-started/gs-normalization-encoding.md

```diff
@@ -43,7 +43,7 @@ To normalize an email address, complete the following steps:
 
 1. Remove leading and trailing spaces.
 2. If there are uppercase characters, convert them to lowercase.
-3. If there is a period (`.`) in the email address (ASCII decimal code 46/UTF-8 hexadecimal code 2E), remove it.<br/>For example, normalize `jane.doe@example.com` to `janedoe@example.com`.
+3. (Conditional) In `gmail.com` addresses only, if there is a period (`.`) in the address (ASCII decimal code 46/UTF-8 hexadecimal code 2E), remove it.<br/>For example, normalize `jane.doe@gmail.com` to `janedoe@gmail.com`.
```

Comment:
per slack comment - should we put the gmail info together?

Reply:
Done, thx.